build/litsearch/getInternalData.py

```diff
@@ -212,6 +212,27 @@ def getRelevantPRData():
     return textForReviewPRs
 
 
+# Get Mt. Sinai data
+def addMtSinaiReviewLinks(df):
+    # Get list of papers reviewed
+    mtSinaiPapersResponse = requests.get("https://api.github.com/repos/ismms-himc/covid-19_sinai_reviews/contents/markdown_files", headers=headers)
+    mtSinaiPapers = json.loads(mtSinaiPapersResponse.text)
+    reviewedDOIs = [str(paper["name"].split(".md")[0]) for paper in mtSinaiPapers]
```

Comment:
This may need to be robust to errors in the upstream filenames.

Reply:
I agree that this would be nice. I added a TODO comment so we don't forget about it. I don't see any easy solutions right now. I checked the file names and didn't see any error (no reviews that didn't match up with their citation) but did find one error in the DOI listed in the actual review (https://github.com/ismms-himc/covid-19_sinai_reviews/pull/13)